--- gcc-4.5.0.orig/libstdc++-v3/configure.host	2009-11-30 15:22:04.000000000 -0800
+++ gcc-4.5.0/libstdc++-v3/configure.host	2010-07-05 15:21:07.000000000 -0700
@@ -235,7 +236,7 @@
   freebsd*)
     os_include_dir="os/bsd/freebsd"
     ;;
-  gnu* | linux* | kfreebsd*-gnu | knetbsd*-gnu)
+  gnu* | linux* | kfreebsd*-gnu | knetbsd*-gnu | *linux-gnu*)
     if [ "$uclibc" = "yes" ]; then
       os_include_dir="os/uclibc"
     else
